looprpc/server.proto

```diff
@@ -78,6 +78,7 @@ message ServerLoopInRequest {
     bytes swap_hash = 2;
     uint64 amt = 3;
     string swap_invoice = 4;
+    bytes last_hop = 5;
```

Comment:
nit: server protos could be updated in its own commit

Reply:
As a reviewer, I never like proto updates in their own commit without context :)